common/test/acceptance/pages/lms/fields.py

```diff
@@ -59,7 +59,7 @@ def icon_for_field(self, field_id, icon_id):
         """
         self.wait_for_field(field_id)
 
-        query = self.q(css='.u-field-{} .u-field-icon'.format(field_id))
+        query = self.q(css='.u-field-{} .u-field-icon .fa'.format(field_id))
```

Comment:
If you remove the "fa" class here, I think it might fix the Bok Choy test. The class "u-field-icon" is specific enough.

With "fa", I think the selector would have to be .u-field-icon.fa (note no space) because they are both classes on the same element.